corehq/apps/enterprise/tests/test_interface.py

```diff
@@ -0,0 +1,262 @@
+from corehq.apps.accounting.tests import generator
+from datetime import datetime
+from decimal import Decimal
+import math
+import uuid
+
+from django.test import TestCase
+from django.test.client import RequestFactory
+
+from corehq.messaging.smsbackends.test.models import SQLTestSMSBackend
+from corehq.apps.smsbillables.models import SmsBillable, SmsGatewayFee
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.users.models import WebUser
+from corehq.apps.sms.models import OUTGOING, SMS
+
+from ..interface import EnterpriseSMSBillablesReport
+
+
+class TestEnterpriseSMSBillablesReport(TestCase):
+    #Tests
+    def test_headers(self):
+        request = RequestFactory().get('')
+        interface = self.create_interface(request)
+        test_headers = [['Date of Message', 'Project Space', 'Direction', 'SMS parts', 'Gateway',
+            'Gateway Charge', 'Usage Charge', 'Total Charge', 'Message Log ID', 'Is Valid?', 'Date Created']]
+        self.assertEqual(interface.headers.as_export_table, test_headers)
+
+    def test_sms_billables_sent_between(self):
+        self.create_smsbillable(datetime(2021, 7, 12))
+        self.create_smsbillable(datetime(2021, 7, 1))
+        self.create_smsbillable(datetime(2021, 5, 5))
+
+        report = self.create_interface(
+            date_sent_startdate='2021-06-30',
+            date_sent_enddate='2021-07-30'
+        )
+
+        results = report.get_all_rows
+
+        self.assertEqual(len(results), 2)
+
+    def test_sms_billables_date_created(self):
```

Comment:
this test isn't testing the created on filter since the created on date is set to the date the test is run and the filtering is actually filtering on the date sent.

Also, this test broke the build once the real date went past 2021-07-30: https://github.com/dimagi/commcare-hq/pull/30172